### DISCUSSION TITLE
Estimated Amount 835 Requirement

### DISCUSSION THREAD

**Discussion by vduffel99** [CMSgov/hospital-price-transparency#107] [Q&A] [answered]
Hi - I was on a webinar today hosted by HFMA. Turquoise Health made a comment that CMS confirmed only 835 files should be used when calculating the Estimated Amount. I have reviewed the Data Dictionary and 45 CFR for this information and there is no reference to what should be used. Can CMS confirm what data should be used to calculate the Estimated Amount? Thank you for all the help

**Comment by RidenourK** [comment, marked as answer]
@vduffel99. Thank you for your question. 

In the CY 2024 OPPS/ASC Final Rule, we indicated we believe hospitals should retain flexibility, in the interest of reducing burden, to determine the best data source for calculating the estimated allowed amount. We therefore declined at the time to be prescriptive. However, we indicated we agree that using information from the EDI 835 electronic remittance advice (ERA) transaction, the electronic transaction that provides claim payment information, including any adjustments made to the claim, such as denials, reductions, or increases in payment, would appear to meet this requirement as the data in the 835 form is used by hospitals to track and analyze their claims and reimbursement patterns. Please refer to [88 FR 82100](https://www.federalregister.gov/d/2023-24293/p-4993).
